calico_containers/calico_ctl/node.py

```diff
@@ -64,6 +65,10 @@
 DEFAULT_IPV4_POOL = IPPool("192.168.0.0/16")
 DEFAULT_IPV6_POOL = IPPool("fd80:24e2:f998:72d6::/64")
 
+KUBERNETES_BINARY_URL = 'https://github.com/Metaswitch/calico-docker/releases/download/v0.5.0/calico_kubernetes'
+KUBERNETES_PLUGIN_DIR = '/usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/'
```

Comment:
As discussed, we should try the backup directory as well (@caseydavenport knows where this is in his k8s fork)